src/pages/home/ReportScreen.js

```diff
@@ -259,7 +260,7 @@ function ReportScreen({
         [route],
     );
 
-    useEffect(() => {
+    useFocusEffect(useCallback(() => {
         const unsubscribeVisibilityListener = Visibility.onVisibilityChange(() => {
             const isTopMostReportID = Navigation.getTopmostReportId() === getReportID(route);
```

Comment:
do we still need this logic if we're unsubscribing when we lose focus?

Reply:
since we will unsubscribe from the event when the component loses focus, do we need to check if the isTopMostReport? Like, this should only be called from the top most report

Reply:
Yeah, I was considering that, but I decided to take the safer approach, since the `Visibility` lib is being used in some of the most crucial components of the app like `Expensify.js`, `Form.js` or `TextInput.js`